custom_components/spook/ectoplasms/cloud/entity.py

```diff
@@ -4,8 +4,9 @@
 
 from typing import TYPE_CHECKING
 
+from homeassistant.components.cloud.client import CloudClient
 from homeassistant.components.cloud.const import DOMAIN as CLOUD_DOMAIN
-from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.device_registry import DeviceInfo
```

Comment:
The location change in core. Will update the remaining cases in another PR. Just this one here to prevent merge conflicts.